modules/standard/IO.chpl

```diff
@@ -1194,6 +1194,7 @@ module IO {
 
 use SysBasic;
 use SysError;
+use Path;
```

Comment:
Alternatively, you can add the `use` statement inside the function itself.  But I would still prefer the function get added to the Path module